templates/influxdb.conf.erb

```diff
@@ -26,9 +26,13 @@ hostname = "<%= @hostname -%>"
 [data]
   dir = "<%= @data_dir -%>"
   max-wal-size = <%= @max_wal_size %>
-  wal-dir = "<%= @wal_dir -%>"
   wal-flush-interval = "<%= @wal_flush_interval -%>"
   wal-partition-flush-delay = "<%= @wal_partition_flush_delay -%>"
+<%- if scope.function_versioncmp([@version, '0.9.3']) >= 0 -%>
```

Comment:
cool. I didn't know I can call puppet functions directly from the template !!! tnx